mycodo/mycodo_flask/routes_method.py

```diff
@@ -58,118 +56,7 @@ def method_data(method_id):
     # User-edited lines of each method
     method_data = MethodData.query.filter(MethodData.method_id == method.unique_id)
 
-    # Retrieve the order to display method data lines
-    display_order = csv_to_list_of_str(method.method_order)
-
-    last_method_data = None
-    if display_order is not None:
-        method_data = MethodData.query.filter(
-            MethodData.method_id == method.unique_id)
-        last_method_data = method_data.filter(
-            MethodData.unique_id == display_order[-1]).first()
```

Comment:
I'm not entirly sure what this code did. last_method_data seems only to be used for method types that only have one entry there. In the AbstractMethod implementation, I reused the method_data_first instead. **Please check if I missed something.**